archive/common/src/main/scala/uk/ac/wellcome/platform/archive/common/progress/models/Progress.scala

```diff
@@ -40,12 +41,13 @@ case object Progress extends URIConverters {
     override def toString: String = failedString
   }
 
-  def apply(createRequest: ProgressCreateRequest): Progress = {
+  def apply(createRequest: RequestDisplayIngest): Progress = {
```

Comment:
Apropos of this, do we want to rename the apps to something less generic? Happy to concede that they should be named for their purpose (which is now `Ingest`).

Reply:
Yeah, I think possibly the names should be revised